packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -645,7 +632,7 @@ const ScrollAreaViewportImpl = React.forwardRef(function ScrollAreaViewportImpl(
       }}
     >
       <div
-        {...(selector ? getSelectorObj(selector + '-position-inner') : undefined)}
+        data-radix-scroll-area-viewport-position-inner=""
```

Comment:
inlined for now but will prob eventually be fixed by exposing one or more elements